### PR TITLE
Caches array length in flattenStyles methods for loop

### PR DESCRIPTION
**Pull request by prometheansacrifice**
Gains minor perf improvement in the for loop by caching the array length

### CODE REVIEW

File: Libraries/StyleSheet/flattenStyle.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ function flattenStyle(style: ?StyleObj): ?Object {
   }
 
   var result = {};
-  for (var i = 0; i < style.length; ++i) {
+  for (var i = 0, styleLength = style.length; i < styleLength; ++i) {
```
Comment (satya164) [inline Libraries/StyleSheet/flattenStyle.js]: Why not just `l`?

**Comment by satya164** [comment]
@facebook-github-bot shipit

File: Libraries/StyleSheet/flattenStyle.js (hunk 1 above)
Comment (prometheansacrifice) [inline Libraries/StyleSheet/flattenStyle.js]: @satya164 Just thought verbosity would be better :)

**Comment by spicyj** [comment]
In the past when we benchmarked caching the lengths everywhere in React vs. never caching the lengths, there was no significant difference so we opted to never cache for simpler code.

**Comment by prometheansacrifice** [comment]
Is it public? Can I run the benchmark locally?

**Comment by spicyj** [comment]
I don't have it handy; it's in an issue or PR comment somewhere on the React repo. You could replicate it yourself, I'm sure.

**Pull request closed without merging**